shared/provision/password/index.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react'
+import * as Styles from '../../styles'
+import * as Kb from '../../common-adapters'
+import {globalColors} from '../../styles'
+import {SignupScreen, errorBanner} from '../../signup/common'
+
+export type Props = {
+  onSubmit: (password: string) => void
+  onBack: () => void
+  onForgotPassword: () => void
+  waiting: boolean
+  error: string
+  username?: string
+}
+
+const Password = (props: Props) => {
+  const [password, setPassword] = React.useState('')
+  const {onSubmit} = props
+  const _onSubmit = React.useCallback(() => onSubmit(password), [password, onSubmit])
+
+  return (
+    <SignupScreen
+      banners={[...errorBanner(props.error)]}
+      buttons={[
+        {
+          disabled: !password,
+          label: 'Continue',
+          onClick: _onSubmit,
+          type: 'Default',
+          waiting: props.waiting,
+        },
+      ]}
+      onBack={props.onBack}
+      title="Enter your password"
+    >
+      <Kb.UserCard
+        style={styles.card}
+        username={props.username}
+        avatarBackgroundStyle={styles.outerCardAvatar}
+        outerStyle={styles.outerCard}
+        lighterPlaceholders={true}
+        avatarSize={96}
+      >
+        <Kb.Box2 direction="vertical" fullWidth={true} style={styles.wrapper} gap="xsmall">
+          <Kb.LabeledInput
+            autoFocus={true}
+            placeholder="Password"
+            onEnterKeyDown={_onSubmit}
+            onChangeText={setPassword}
+            value={password}
+            textType="BodySemibold"
+            type="password"
+          />
+          <Kb.Text
```

Comment:
Impossible to tap this on a 5S-sized screen
<img width="450" alt="image" src="https://user-images.githubusercontent.com/11968340/64380419-ef99fd00-cffe-11e9-85e6-db36e1ebcd9d.png">